app/controllers/comments_controller.rb

```diff
@@ -13,16 +13,21 @@ def show
   # GET /comments/new
```

Comment:
You could get rid of the redundancy of the authorize statements and just do:  before_action {authorize @comment || Comment }